exp/term/ansi/wrap.go

```diff
@@ -199,6 +202,8 @@ func Wordwrap(s string, limit int, breakpoints string) string {
 			case unicode.IsSpace(r):
 				addWord()
 				space.WriteByte(b[i])
+			case r == '-':
```

Comment:
Would there be value in having a small helper function `IsDash` to be consistent to how it is done for `IsSpace`?

Reply:
Perhaps, if we add support to other unicode dash characters 🤔

Reply:
Something to consider, certainly nothing to stop you merging once you do decide.

Reply:
Thinking more about this, there are many variants of dashes defined in unicode. People who wish to use custom breakpoints can do so using the `breakpoints` parameter